lib/rubocop/git/commit_file.rb

```diff
@@ -29,7 +32,7 @@ def removed?
   end
 
   def ruby?
-    filename.match(/.*\.(rb|rake)|Gemfile|Rakefile$/)
+    filename.match(/.*\.(#{RUBY_EXT.join('|')})|#{RUBY_FILES.join('$|')}/)
```

Comment:
I know I gave it as an example, but in the end we're building both inefficient and hard to read regexp here. Maybe we can skip it for the known ruby files?

```ruby
filename.match(/.*\.(rb|rake)/) || KNOWN_RUBY_FILES.include?(filename)
```